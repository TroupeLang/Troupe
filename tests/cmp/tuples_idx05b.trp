(1,2,3).2147483648