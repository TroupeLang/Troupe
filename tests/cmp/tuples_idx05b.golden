troupec: ProjIdx: illegal index: 2147483648 (max index: 2147483647)
